addons/stock/tests/__init__.py

```diff
@@ -4,3 +4,4 @@
 import test_product
 import test_warehouse
 import test_shipment
+from . import test_stock_location_search
```

Comment:
@moylop260 since all other tests are imported without `from .` I was thinking it would be cleaner to keep them all the same, since otherwise you have both `import test_product` and `from . import test_stock_location_search`.